guides/shipments/overview-of-shipments.md

```diff
@@ -0,0 +1,463 @@
+
+### Zones
+
+Zones serve as a mechanism for grouping geographic areas together into a single
+entity.
+
+The shipping address entered during checkout defines the zone the customer is
+in. This limit the available shipping methods and defines regional taxation
+rules.
+
+<!-- TODO:
+  For more information about zones, see [the Locations guide](../locations).
+-->
+
+### Shipping calculators
+
+A shipping calculator is the component responsible for calculating the shipping
+rate for each available shipping method.
+
+Solidus ships with five default shipping calculators:
+
+- [Flat percent](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/shipping/flat_percent_item_total.rb)
+- [Flat rate (per order)](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/shipping/flat_rate.rb)
+- [Flat rate per package item](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/shipping/per_item.rb)
+- [Flexible rate per package item](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/shipping/flexi_rate.rb)
+- [Price sack](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/shipping/price_sack.rb)
+  (which offers variable shipping cost that depends on order total)
+
+Flexible rate is defined as a flat rate for the first product, plus a different
+flat rate for each additional product.
+
+If you want to estimate shipping rates using carrier APIs, You can use an
+extension like [`solidus_active_shipping`][solidus-active-shipping]. Or, if you
+have other complex needs, you can create a [custom shipping
+calculators][custom-shipping-calculators] for more information.
+
+### Shipping rates
+
+For each shipment, a `Spree::ShippingRate` object is created for each of your
+store's shipping methods. These objects calculate how much the shipment would
+cost if it were sent by each of the available shipping methods:
+
+| `Spree::ShippingRate` | 1     | 2     | 3     | Description                                         |
+|-----------------------|-------|-------|-------|-----------------------------------------------------|
+| `shipment_id`         | 1     | 1     | 1     | All of these rates are for one shipment             |
+| `shipping_method_id`  | 1     | 2     | 3     | Each available shipping method                      |
+| `selected`            | false | true  | false | Set to `true` for the selected shipping method only |
+| `cost`                | $0.50 | $1.75 | $1.25 | The shipment's shipping rate                        |
+
+Once the shipping method has been chosen, the matching `Spree::ShippingRate`'s
+`selected` key becomes `true`. Only one shipping rate can be `true` for each
+shipment.
+
+[solidus-active-shipping]: solidus-active-shipping-extension.html.markdown
+
+[custom-shipping-calculators]: custom-shipping-calculators.html.markdown
+
+## Shipment setup examples
+
+### Simple setup
```

Comment:
Can we avoid exceed the 80-char limit. Of course we can do that when we are sure the content is ready to be merged, just wanted to point that out so we don't forget. :)

Reply:
Absolutely. Thanks for noting this.

Right now, all of the legacy content uses >80 character lines. I will be pruning these sections in further pull requests. (I'm just superficially breaking up these PRs since there is _a lot_ of content.)